android-o/DownloadableFonts/DownloadableFonts/QueryBuilder.cs

```diff
@@ -0,0 +1,93 @@
+﻿using Java.Lang;
+
+namespace DownloadableFonts
+{
+	public class QueryBuilder
+	{
+		String FamilyName = null;
+
+		Float Width = null;
+
+		Integer Weight = null;
+
+		Float Italic = null;
+
+		Java.Lang.Boolean Besteffort = null;
```

Comment:
Can use a bool (System.Boolean) here and avoid the need to construct a new Java.Lang.Boolean instance later.

Reply:
@pjcollins Yes, but I don't know how to check if that property was assigned or not. Required here: https://github.com/xamarin/monodroid-samples/pull/203/files/db1794b67fa36cefda92307ee31cce11dc2123ac#diff-adade3c305f996bafb2c367743eb5490R85

Reply:
Ahh, using `bool?` will allow it to be nullable. 
https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/nullable-types/using-nullable-types

Reply:
Nice! I thought it from Java perspective 👍